testsuite/settings.py

```diff
@@ -12,12 +12,13 @@
     }
 }
 
-CONSUL_URL = os.environ.get('TESTING_CONSUL_URL', 'http://localhost:8500')
-NOMAD_URL = os.environ.get('TESTING_NOMAD_URL', 'http://localhost:4646')
+CONSUL_URL = os.environ.get('TESTING_CONSUL_URL', 'http://10.66.60.1:8500')
+NOMAD_URL = os.environ.get('TESTING_NOMAD_URL', 'http://10.66.60.1:4646')
 NOMAD_JOB_PREFIX = f'testsuite-{random_code(8)}-'
 NOMAD_DEPLOYMENT_NAME = f"test {datetime.now().strftime('%H:%M:%S')}"
 
 VMCK_BACKEND = os.environ.get('TESTING_BACKEND', 'docker')
 
 _github_image = 'https://github.com/mgax/vmck-images/raw/master/bionic.qcow2'
-QEMU_IMAGE_URL = os.environ.get('TESTING_QEMU_IMAGE_URL', _github_image)
+QEMU_IMAGE_PATH_PREFIX = os.environ.get('TESTING_QEMU_IMAGE_PATH_PREFIX',
+                                        _github_image)
```

Comment:
Yeah this doesn't make sense any more. You're using the same github image URL but now it's for a prefix instead of a file. I think at this point you should drop the default value and make sure the docs explain how to download and set up an image.